FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Prabhat Tiwari on 20/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
```

Comment:
Same as my previous comment
```suggestion
final class ManagedFeedImage: NSManagedObject {
```